main.go

```diff
@@ -182,7 +182,10 @@ func (s *prometheusServer) getPrometheusTLSConfig(ctx context.Context, certWatch
 }
 
 func newPrometheusServer(metricsAddr string, cache cache.Cache) *prometheusServer {
-	sspMetrics.SetupMetrics()
+	err := sspMetrics.SetupMetrics()
+	if err != nil {
+		panic(err)
+	}
```

Comment:
I see they have an error handling after the usage of this function, you can actually return an error here instead of panic, and let the caller handle it

Reply:
I fixed it but not sure this is how you meant so if you can take a look (:

Reply:
that's it, just remove the else, you are already returning in the if, so you can just return after that

Reply:
right! thanks fixed